crates/lib3h/src/gateway/gateway_dht.rs

```diff
@@ -102,28 +104,13 @@ impl<T: Transport, D: Dht> P2pGateway<T, D> {
                     p2p_gossip
                         .serialize(&mut Serializer::new(&mut payload))
                         .unwrap();
```

Comment:
Do you think that we can have an `expect` here instead of a `unwrap`? (it's most of the time easier to debug later IMHO :) )